### PR TITLE
add embark theme

### PR DESCRIPTION
**Pull request by gennaro-tedesco**

### CODE REVIEW

File: examples/embark.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "page": "#1e1c31",
+  "text": "#8a889d",
+  "background": "#3E3859",
+  "highlight": "#62d196",
```
Comment (skbolton) [inline examples/embark.json]: Can we make this the other green color #A1EFD3

File: examples/embark.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  "highlight_active": "#f2b482",
```
Comment (skbolton) [inline examples/embark.json]: For the highlight can we do #FFE6B3

File: examples/embark.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "page": "#1e1c31",
+  "text": "#8a889d",
```
Comment (skbolton) [inline examples/embark.json]: I worry about the contrast with this can it be changed to #CBE3E7

Comment (gennaro-tedesco) [reply examples/embark.json]: I modifed the colours according to your suggestions: it looks like the following now

Screenshot 2022-10-17 at 19 50 41
Screenshot 2022-10-17 at 19 50 50

File: examples/embark.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+  "page": "#1e1c31",
+  "text": "#cbe3e7",
+  "background": "#3E3859",
+  "highlight": "#a1efd3",
```
Comment (skbolton) [inline examples/embark.json]: Looking much better! Sorry I realize now how the highlight and highlight active highlights work. The active should be the lighter color #FFE6B3 and the non active the darker #F2B482.

**Review by skbolton — COMMENTED** [review]
Final adjustment and then I think we are good.

File: examples/embark.json (hunk 4 above)
Comment (gennaro-tedesco) [reply examples/embark.json]: This is how it is at the moment though, with active being in fact `#FFE6B3`, correct?

Comment (skbolton) [reply examples/embark.json]: Highlight is set to `#a1efd3` instead of `#F2B482`.

Comment (gennaro-tedesco) [reply examples/embark.json]: Fixed.

**Pull request merged**